lib/puppet/provider/package/rpm.rb

```diff
@@ -120,6 +120,8 @@ def install
 
     # RPM gets upset if you try to install an already installed package
     return if @resource.should(:ensure) == version || (@resource.should(:ensure) == :latest && version == latest)
+    # Do now downgrade the package if :dowgrade = false
```

Comment:
Yeah I noticed that too right after the PR. I fixed it post `git reset HEAD~` and the typo will be gone in the next force push.